src/components/CallView/Grid/Grid.vue

```diff
@@ -355,6 +361,16 @@ export default {
 		showNavigation() {
 			return this.gridWidth > 0 && this.isStripe && this.videosCount > 0 && this.showVideoOverlay
 		},
+
+		// Blur radius for each bacground in the grid
+		videoBackgroundBlur() {
+			// The amount of blur
+			const amount = this.$store.getters.videoBackgroundBlur
+			// Represents the surface of the element
+			const surfaceMultiplier = (this.videoWidth * this.videoHeight) / 100
+			// Calculate the blur
+			return `filter: blur(${surfaceMultiplier * amount}px)`
```

Comment:
extract the math to another computed and add the blur factor to the grid dev details view?